doc/TBG_macros.cfg

```diff
@@ -186,7 +186,8 @@ TBG_checkpoints="--checkpoints 1000"
 
 # Restart the simulation from checkpoints created using TBG_checkpoints
 TBG_restart="--restart --restart-step 1000"
```

Comment:
have you tested that it still works and finds the *latest* checkpoint automatically?